README.md

```diff
@@ -298,7 +298,15 @@ element.getAttribute("attribute")
 </pre>
 
 <pre>
-element.scrollIntoView()
+element.scrollTo()
+</pre>
+
+<pre>
+element.scrollUp()
```

Comment:
I think it would be a nice idea to point out that both methods have the same functionality for the web version, as it uses the same selenide method